CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ changes in the following format: PR #1234***
 - Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
 - Unix user permissions have been updated which may affect access to files. New
 documentation for file permissions has been added to the README.md file (PR #5323)
+- Very old instrument relying on QuickForm may have issues due to code changes (PR #4928)
+- Module-specific permissions added for Survey Accounts, Imaging Behavioural
```

Comment:
This seems to be rebased incorrectly and is duplicating a line. Also, this is in the wrong version of LORIS in the release notes.. it's going to master, so it won't be in 23.0